server/src/test-fast/java/com/thoughtworks/go/server/service/StageManualTriggerCheckerTest.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2019 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.thoughtworks.go.server.service;
+
+import com.thoughtworks.go.domain.Pipeline;
+import com.thoughtworks.go.server.service.result.OperationResult;
+import com.thoughtworks.go.serverhealth.HealthStateType;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static com.thoughtworks.go.server.service.ScheduleStageResult.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+
+public class StageManualTriggerCheckerTest {
+
+    private SchedulingCheckerService schedulingCheckerService;
+    private PipelineService pipelineService;
+    private Pipeline pipeline;
+    private String pipelineName;
+    private StageManualTriggerChecker checker;
+    private String stageName;
+    private OperationResult result;
+    private int pipelineCounter;
+
+    @BeforeEach
+    void setUp() throws Exception {
+        schedulingCheckerService = mock(SchedulingCheckerService.class);
+        pipelineService = mock(PipelineService.class);
+        pipeline = mock(Pipeline.class);
```

Comment:
Why do you want to mock a POJO?

Reply:
It can be removed. Will make that change.